BenadrylAutomation/src/main/java/com/jnj/pageobjects/DifferencesPage.java

```diff
@@ -0,0 +1,181 @@
+			Action.verifyPageUrl(expectedUrl);
+		} else
+			extentFailLog("Header displayed : ", false);
+	}
+
+	/**
+	 * Function to verify article links
+	 * 
+	 * @param menu        pass menu text
+	 * @param headerTitle pass header title
+	 * @param article     pass article
+	 * @param readMore    pass read more link
+	 * @param expectedUrl pass expected URL
+	 */
+	public void verifyArticles(String menu, String headerTitle, String article, String readMore, String expectedUrl) {
+		Action.explicitWaitForElementTobeclickable(header.getMenuItem(menu), 30);
+		Action.performActionwithExtentInfoLog(header.getMenuItem(menu), "click",
+				"Clicking on : " + header.getMenuItem(menu).getText());
+		Action.explicitWait(relatedContentHeader, 30);
+		boolean eleDisplayed = relatedContentHeader.isDisplayed();
+		if (eleDisplayed) {
+			extentPassLog("Related Content header displayed : ", true);
+			String actualHeader = relatedContentHeader.getText();
+			Action.printAndAssert(actualHeader, headerTitle);
+			Action.explicitWait(articleName(article), 30);
+			String actualArticle = articleName(article).getText();
+			Action.printAndAssert(actualArticle, article);
+			Action.explicitWait(readMoreLink(readMore), 30);
+			readMoreLink(readMore).click();
+			extentInfoLog("Clicked on : ", "READ MORE link");
+			Action.verifyPageUrl(expectedUrl);
+		} else
+			extentFailLog("Related Content header displayed : ", false);
+	}
+
+	/**
+	 * Function to verify related product
+	 * 
+	 * @param menu            pass menu text
+	 * @param heading         pass header heading
+	 * @param product         pass product
+	 * @param expectedUrl     pass expected URL
+	 * @param ecpectedAges    pass expected ages
+	 * @param expectedProduct pass expected product
+	 * 
+	 */
+	public void verifyRelatedProducts(String menu, String heading, String product, String expectedUrl,
+			String expectedAges, String expectedProduct) {
+		Action.explicitWaitForElementTobeclickable(header.getMenuItem(menu), 30);
+		Action.performActionwithExtentInfoLog(header.getMenuItem(menu), "click",
+				"Clicking on : " + header.getMenuItem(menu).getText());
+
+		Action.explicitWait(articleName(heading), 30);
+		boolean eleDisplayed = articleName(heading).isDisplayed();
+		if (eleDisplayed) {
+			extentPassLog("Related Products header displayed : ", true);
+			String actualArticle = articleName(heading).getText();
+			Action.printAndAssert(actualArticle, heading);
+			Action.explicitWait(productName(product), 30);
+			productName(product).click();
+			extentInfoLog("Clicked on : ", product);
+			Action.verifyPageUrl(expectedUrl);
+			Action.navigateBack();
+			extentInfoLog("Navigating back to : ", "Difference page");
+			Action.explicitWait(getAges(product), 30);
+			String actualAges = getAges(product).getText();
+			Action.printAndAssert(actualAges, expectedAges);
+			Action.explicitWait(buyNowBtn(product), 30);
+			buyNowBtn(product).click();
+			extentInfoLog("Clicked on : ", "buy now button");
```

Comment:
replace the click and log with "Action.performActionwithExtentInfoLog"
@vaibhavNagvekarSJI

Reply:
fixed